apps/gitness/src/pages-v2/repo/transform-utils/branch-transform.ts

```diff
@@ -0,0 +1,29 @@
+import { TypesBranchExtended, TypesCommitDivergence } from '@harnessio/code-service-client'
+
+import { timeAgoFromISOTime } from '../../../pages/pipeline-edit/utils/time-utils'
+
+export const transformBranchList = (
```

Comment:
i think we should add a return type on this function. it should be available to be imported from the `ui` package

Reply:
yes I can add the return type.